openedx/features/edly/tests/test_models.py

```diff
@@ -0,0 +1,102 @@
+    Tests for Organization Model.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super(OrganizationTests, cls).setUpClass()
+
+    def test_organization_post_save_receiver(self):
+        """
+        Test entry is added to organization model and filter on it's value.
+        """
+        # add Organization entry to database
+        org_name = 'test org name'
+        organization = EdlyOrganizationFactory(name=org_name)
+
+        # Verify an entry to Organization was added.
+        organizations = EdlyOrganization.objects.filter(name=org_name).all()
+
+        # Make sure an entry (and only one entry) is saved for Organization
+        assert len(organizations) == 1
+
+    def test_site_configuration_post_update_receiver(self):
+        """
+        Test that entry is updated in Organization model.
+        """
+
+        org_name = 'updated test org name'
+
+        # add SiteConfiguration to database
+        organization = EdlyOrganizationFactory()
+
+        organization.name = org_name
+        organization.save()
+
+        # Verify an entry to Organization was updated.
+        organizations = EdlyOrganization.objects.filter(name=org_name).all()
+
+        # Verify an entry to Organization was updated is filtered
+        assert organizations[0].name == org_name
+        assert len(organizations) == 1
+
+
+class SubOrganizationTests(TestCase):
+    """
+    Tests for SubOrganization Model.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super(SubOrganizationTests, cls).setUpClass()
+        cls.lms_site = SiteFactory()
+        cls.studio_site = SiteFactory()
+
+    def test_sub_organization_post_save_receiver(self):
+        """
+        Test that entry is added to SubOrganization model.
+        """
+        # add SubOrganization to database
+        sub_organization = EdlySubOrganizationFactory.create(lms_site=self.lms_site)
+
+        # Verify an entry to SubOrganization was added.
+        sub_organization_data = EdlySubOrganization.objects.filter(lms_site=sub_organization.lms_site).all()
+
+        # Make sure an entry (and only one entry) is saved for SubOrganization
+        assert len(sub_organization_data) == 1
+
+    def test_sub_organization_post_update_receiver(self):
+        """
+        Test that entry is updated in SubOrganization model.
+        """
+
+        org_name = 'updated test org name'
+
+        # add SiteConfiguration to database
+        edly_organization = EdlyOrganizationFactory()
+
+        edly_organization.name = org_name
+        edly_organization.save()
+
+        # add SubOrganization to database
+        sub_organization = EdlySubOrganizationFactory.create(studio_site=self.studio_site)
+
+        sub_organization.edly_organization = edly_organization
+        sub_organization.save()
+
+        # Verify an entry to SubOrganization was updated.
+        sub_organization_data = EdlySubOrganization.objects.filter(studio_site=sub_organization.studio_site).all()
```

Comment:
I don't think we need to an `.all()` here. the filter already fetches all the filtered values.

Reply:
You are right. Since `Entry.objects.filter` is same `Entry.objects.all().filter`. Feedback implemented